.rubocop_todo.yml

```diff
@@ -18,10 +18,11 @@ Capybara/SpecificFinders:
   Exclude:
     - 'spec/features/embed_this_panel_spec.rb'
 
-# Offense count: 1
+# Offense count: 2
 # Configuration parameters: AllowedMethods, AllowedPatterns, CountRepeatedAttributes, Max.
 Metrics/AbcSize:
   Exclude:
+    - 'lib/bcp47/registry.rb'
```

Comment:
Would you please move these to inline exclusions rather than adding to the TODO?

Reply:
Sure.